cinnamon/vizceral/README.md

```diff
@@ -0,0 +1,36 @@
+# Troubleshooting Vizceral
+
+If you get stuck at the loading screen, there are a couple of things to check.
+
+First, are all the right containers running.
+
+```
+> docker ps -a --format "{{.Names}}" | grep cinnamonsamplevzc
+cinnamonsamplevzc_web_1
+cinnamonsamplevzc_backend_1
+cinnamonsamplevzc_elasticsearch_1
+```
+
+If any of the containers are not running, then CTRL-C the `docker-compose command` and make sure that all containers are stopped and removed.
+
+```
+> docker ps -a --format "{{.Names}}" | grep cinnamonsamplevzc | xargs docker stop
+> docker ps -a --format "{{.Names}}" | grep cinnamonsamplevzc | xargs docker rm
```

Comment:
Maybe add a `-f` to the end of the `docker rm` command?

Reply:
Force all the things...

Reply:
Use the force as someone said :)